ndpi-netfilter/ipt/libxt_ndpi.c

```diff
@@ -800,9 +800,10 @@ void _init(void)
 			   vs = strstr(buf,"#version");
 			   if(!vs)
 	    			xtables_error(PARAMETER_PROBLEM, "xt_ndpi: #version missing");
-			   if(!strstr(vs+8,NDPI_GIT_RELEASE))
-	    			xtables_error(PARAMETER_PROBLEM, "xt_ndpi: module version %s != %s",
-						vs+8,NDPI_GIT_RELEASE);
+			   // TODO - skipping ndpi version
```

Comment:
@DeepakDoddigarla-Cyberreef , why is this change needed?

Reply:
While adding iptables-ndpi rules, it throws me error and commenting this part helped me to add rule. I don't think we need an internal build check (which changes frequently)

Reply:
I think we should figure out how to fix this.